teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<p>* Protein Data Bank</p><p>* Protein-ligand complex</p><p>* documentation on Sourceforge</p><p>* 3rd question could be more specific. Ask for a particular part of the command-line. Maybe a minimization only function, or how to handle parallelization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>